app/scripts/components/dashboardComponents/breadcrumbs.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react'
```

Comment:
You'll want to remove the PropTypes import from React. The package is now separate and it looks like you have both listed here.

Reply:
Mostly Notes for myself
Push up fixes for this